### ISSUE TITLE
trailingCommas rule gets applied to a dictionary's type declaration

### ISSUE THREAD

**Issue by csimmons0** [nicklockwood/SwiftFormat#89]
Unless I disable the trailingCommas rule, this code:
```
    static let FlattenedScheduleItemAppearanceMap: [FlattenedDoctorScheduleItemType:
                                                    DoctorSetAvailabilityTimeCellAppearance] = [
        .BookedTimeBlock: .GrayedOut,
        .AvailableTimeBlock: .Checked,
        .ClosedTimeBlock: .Normal
    ]
```
gets converted to this invalid code:
```
    static let FlattenedScheduleItemAppearanceMap: [
        FlattenedDoctorScheduleItemType:
            DoctorSetAvailabilityTimeCellAppearance,
    ] = [
        .BookedTimeBlock: .GrayedOut,
        .AvailableTimeBlock: .Checked,
        .ClosedTimeBlock: .Normal,
    ]
```
This bug doesn't always apply to dictionaries. The type declaration isn't modified when swiftformat runs on the following code:
```
    static let FlattenedScheduleItemAppearanceMap:
        [FlattenedDoctorScheduleItemType: DoctorSetAvailabilityTimeCellAppearance] = [
            .BookedTimeBlock: .GrayedOut,
            .AvailableTimeBlock: .Checked,
            .ClosedTimeBlock: .Normal
        ]
```

**Comment by nicklockwood** [comment]
Thanks, I'll take a look.

**Comment by nicklockwood** [comment]
Fixed in 0.24.5. Thanks for the report!
